cmd/vl3-nse/aws_tgw_connect.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"context"
+
+	"github.com/golang/protobuf/ptypes/empty"
+	"github.com/sirupsen/logrus"
+
+	"github.com/cisco-app-networking/nsm-nse/pkg/nseconfig"
+	"github.com/networkservicemesh/networkservicemesh/controlplane/api/connection"
+	"github.com/networkservicemesh/networkservicemesh/controlplane/api/networkservice"
+	"github.com/networkservicemesh/networkservicemesh/sdk/endpoint"
+)
+
+type AwsTgwConnector struct {
+	TransitGatewayID 	string
+	TransitGatewayName 	string
+}
+
+func newAwsTgwConnector(ate *nseconfig.AwsTgwEndpoint) *AwsTgwConnector {
+	logrus.Infof("newAwsTgwConnector()")
+
+	if ate != nil {
+		newAwsTgwConnector := &AwsTgwConnector{
+			TransitGatewayID:   ate.TransitGatewayID,
+			TransitGatewayName: ate.TransitGatewayName,
+		}
+
+		logrus.Infof("newAwsTgwConnector returning object: %v", newAwsTgwConnector)
+		return newAwsTgwConnector
+	}
+
+	logrus.Infof("newAwsTgwConnector returning")
```

Comment:
I think you can be more explicit about this log.

Reply:
Sure